src/odemis/gui/util/img.py

```diff
@@ -860,24 +860,28 @@ def calculate_ticks(value_range, client_size, orientation, tick_spacing):
     vtp_ratio = abs(pixel_space / value_space)  # px/unit
     epsilon = (1 / vtp_ratio) / 10  # "tiny" value: a 10th of a pixel
 
-    num_ticks = pixel_space // tick_spacing
+    # Find the ticks to show so that it looks good for the user. It must have a
+    # constant spacing between the ticks, and this spacing should be a "easy
+    # number": 1, 2, 5, 7.5, or any of these values multiplied by a power of 10.
```

Comment:
Why is 7.5  "easy" and 2.5 not? Is 250 also not fine? or 300?
And what does "easy" mean? Why is it easy?
Can you maybe add what easy means? Probably it has something to do with the layout?

Reply:
To be honest, I'm not exactly sure on what basis these values where picked, but they were defined by Toon as the simplest distances to have in a graph. I guess the idea is to have only a few distances possible, and they should divide 10. Indeed, 7.5 is a little odd, but I think that jumping from 5 to 10 brings a whole set of graphs with too many ticks. So it's the less bad option. "Easy" here means "easy to the eyes of someone who has a PhD in Physics" ;-) It might sound a little exclusive, but I think that actually defines most of our users ;-) Honestly... it's very arguable... but I'd rather not change it now.

Reply:
Indeed, Toon picked those values. Initially, this code was implemented to start with spacing 10 (multiplied by a power of 10). This spacing would be divided by 2 until the optimal spacing is reached and consequently the final values for spacing would be one of the following: 1.25, 2.5, 5, 10 (multiplied by a power of 10), which are not the desirable ones. I think it's not a big issue, we could again ask for his feedback and change them anytime as far as they remain hardcoded.